### PR TITLE
voc2coco and divide dataset

### PR DESCRIPTION
**Pull request by ytzfhqs**
Support Pascal voc data format to coco data format. Divide dataset into training set, validation set and test set.

## Modification
The voc2coco.py script is provided for Pascal voc data format to coco data format. 
The coco_train_split.py script is provided for dataset splitting.
The README.md file is provided to explain the use of the script.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMPreTrain.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by hhaAndroid** [comment]
@ytzfhqs 

`tools/dataset_converters/pascal_voc.py`
`tools/misc/split_coco.py`

mmdet already has these two features. You can confirm if they meet your requirements.

**Comment by ytzfhqs** [comment]
@hhaAndroid Okay, I'm going to close this PR
